reactor-netty-http/src/main/java/reactor/netty/http/server/HttpServer.java

```diff
@@ -580,6 +580,10 @@ else if (configuration().metricsRecorder() != null) {
 		}
 	}
 
+	public final HttpServer metrics(boolean enable, String prefix) {
+		return metrics(enable, () -> new MicrometerHttpServerMetricsRecorder(prefix));
+	}
+
```

Comment:
Is this needed? One can use

- `reactor.netty.http.server.HttpServer#metrics(boolean, java.util.function.Supplier<? extends reactor.netty.channel.ChannelMetricsRecorder>)`
- `reactor.netty.http.server.HttpServer#metrics(boolean, java.util.function.Supplier<? extends reactor.netty.channel.ChannelMetricsRecorder>, java.util.function.Function<java.lang.String,java.lang.String>)` with `MicrometerHttpServerMetricsRecorder`